### DISCUSSION TITLE
https://cucoders.dev/publicaciones/aprezcuba24/paas1/

### DISCUSSION THREAD

**Discussion by giscus** [CuCodersCommunity/cucoderscommunity.github.io#94] [Page Comments]
# https://cucoders.dev/publicaciones/aprezcuba24/paas1/

En la actualidad se han popularizado el uso de PAAS (Platform as a Service), exisitiendo varias alternativas como heroku, vercel, netlify, etc. En una serie de artículos me propongo implementar mi propio PAAS. Haciendo una prueba de concepto e integrando varias tecnologías libres disponibles para la comunidad.

https://cucoders.dev/publicaciones/aprezcuba24/paas1/

**Comment by manuelernestog** [comment]
Brothe esto que quieres hacer es tremendo proyecto. 👏👏👏

**Reply by aprezcuba24** [reply]
Gracias. En principio va a ser una prueba de concepto con el objetivo de validad la idea y recibir feedback. 

PD: Muchas gracias por crear **cucoders** es un buen lugar para intercambiar lo desarrolladores cubanos.

**Comment by YuniGoldenBoY30** [comment]
Éxitos 🚀🚀🇨🇺
